app/models/manual_payment_record.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class ManualPaymentRecord < ApplicationRecord
+  WCA_TO_MANUAL_PAYMENT_STATUS_MAP = {
+    created: %w[created],
+    succeeded: %w[succeeded],
+  }.freeze
+
+  has_one :registration_payment, as: :receipt
+  has_one :payment_intent, as: :payment_record
+
+  def determine_wca_status
+    WCA_TO_MANUAL_PAYMENT_STATUS_MAP[status.to_sym]
```

Comment:
This used to have the same contents as `status` - I've changed it to pull the wca_status from the defined mapping